File: Python/tests/test_bart_ewmv.py
import pytest

from hbayesdm.models import bart_ewmv


def test_bart_ewmv():
    _ = bart_ewmv(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_ewmv.R
context("Test bart_ewmv")
library(hBayesDM)

test_that("Test bart_ewmv", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_ewmv(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
